### PR TITLE
chore(zoe): add test of burnInvitation.js

### PR DESCRIPTION
**Pull request by katelynsills**
This PR is very small and makes no changes to Zoe src. It adds a test for the file `burnInvitation.js`.

### CODE REVIEW

File: packages/zoe/test/unitTests/zoe/test-burnInvitation.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// @ts-check
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava';
+
+import { makeIssuerKit, AssetKind, AmountMath } from '@agoric/ertp';
+
+import { burnInvitation } from '../../../src/zoeService/offer/burnInvitation';
+
+test('burnInvitation', async t => {
```
Comment (Chris-Hibbert) [inline packages/zoe/test/unitTests/zoe/test-burnInvitation.js:11]: Would you mind adding a test with multiple invitations?

Comment (katelynsills) [reply packages/zoe/test/unitTests/zoe/test-burnInvitation.js:11]: Sure thing

Comment (katelynsills) [reply packages/zoe/test/unitTests/zoe/test-burnInvitation.js:11]: Just added a test that tries to burn multiple invitations at once. Let me know if you meant something else!

**Review by Chris-Hibbert — APPROVED** [review]
Sorry, I had written my response, but failed to submit the review.

**Pull request merged**